pyproject.toml

```diff
@@ -23,16 +23,14 @@ kedro = ">=0.18.0"
 pre-commit = { version = "*", optional = true }
 pytest = { version = ">=5.0", optional = true }
 pytest-cov = { version = "2.10.1", optional = true }
-neptune-optuna = { version = "*", optional = true }
-neptune-fastai = { version = "*", optional = true }
+backoff = { version = "*", optional = true }
```

Comment:
What is this change about?

Reply:
We're getting rid of using external integrations in the sample project (there were both Optuna and Fast.ai). Fast.ai has some issues/warnings in the context of multiprocessing and it shadows problems related only to Kedro integration itself. Backoff is mainly to handle retry instead of sleeping some about in waiting for Leaderboard refresh.